pkg/render/node.go

```diff
@@ -1656,7 +1656,12 @@ func (c *nodeComponent) nodeEnvVars() []corev1.EnvVar {
 	switch c.cfg.Installation.KubernetesProvider {
 	case operatorv1.ProviderOpenShift:
 		// For Openshift, we need special configuration since our default port is already in use.
-		nodeEnv = append(nodeEnv, corev1.EnvVar{Name: "FELIX_HEALTHPORT", Value: "9199"})
```

Comment:
Interesting, I wonder if this was already encountering another bu which was that it wasn't respecting `c.cfg.FelixHealthPort` 

> 		{Name: "FELIX_HEALTHPORT", Value: fmt.Sprintf("%d", c.cfg.FelixHealthPort)},

@tmjd or @mgleung might know better?

To me, it seems like we should probably be addressing this earlier in this function and accounting for the user specified health port - otherwise we're overwriting user input in this branch.